client/keyspace_client.go

```diff
@@ -153,3 +155,35 @@ func (c *client) UpdateKeyspaceState(ctx context.Context, id uint32, state keysp
 
 	return resp.Keyspace, nil
 }
+
+// GetAllKeyspaces get all keyspaces metadata.
+func (c *client) GetAllKeyspaces(ctx context.Context, startID uint32, limit uint32) ([]*keyspacepb.KeyspaceMeta, error) {
```

Comment:
If replace `WatchKeyspaces` with `GetAllKeyspaces`, will it cost too much? Do we need to use watch loop in server?